x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -36,6 +37,37 @@ import {
 const defaultFields = {
   name: '',
   seeds: [],
+  skipUnavailable: false,
+};
+
+const skipUnavailableOptions = [
+  {
+    id: 'skipUnavailableOptionFalse',
+    label: (
+      <FormattedMessage
+        id="xpack.remoteClusters.remoteClusterForm.fieldSkipUnavailable.falseOptionLabel"
+        defaultMessage="No"
+      />
+    ),
+  }, {
+    id: 'skipUnavailableOptionTrue',
+    label: (
+      <FormattedMessage
+        id="xpack.remoteClusters.remoteClusterForm.fieldSkipUnavailable.trueOptionLabel"
+        defaultMessage="Yes"
+      />
+    ),
+  },
+];
+
+const skipUnavailableOptionIdToValueMap = {
```

Comment:
This is a really interesting idea... I spent a good amount of time just now thinking about it, and what the implications would be. To be honest I think I would still use a map to be explicit about how the options map to values, because this is a bit too implicit for me:

```js
skipUnavailable: skipUnavailable.includes('True')
```

And if maps are still used, then this change just means moving the conversion from one place to another. If that's the case, then I think I would prefer to retain the "integrity" of the `fields` object as a consistent representation of the remote cluster, regardless of where in the wizard it's being referenced.